src/cloud-api-adaptor/podvm/qcow2/build-s390x-se-image.sh

```diff
@@ -190,13 +204,19 @@ echo "Generating an IBM Secure Execution image"
 echo "Creating SE boot image"
 export SE_PARMLINE="root=/dev/mapper/$LUKS_NAME rd.auto=1 rd.retry=30 console=ttysclp0 quiet panic=0 rd.shell=0 blacklist=virtio_rng swiotlb=262144"
 sudo -E bash -c 'echo "${SE_PARMLINE}" > ${dst_mnt}/boot/parmfile'
-sudo -E /usr/bin/genprotimg \
+if [ "${SE_VERIFY}" = "true" ]; then
+	sudo -E /usr/bin/genprotimg ${host_keys} \
+--output=${dst_mnt}/boot-se/se.img --image=${dst_mnt}/boot/${KERNEL_FILE} --ramdisk=${dst_mnt}/boot/${INITRD_FILE} \
+--cert=${cacert} --cert=${signcert} --crl=${crl} --parmfile=${dst_mnt}/boot/parmfile
```

Comment:
Can we reduce the duplicate by adding `--cert=${cacert} --cert=${signcert} --crl=${crl}` as extra options in the if block and calling genprotimg outside of it, if these are the only differences?

Reply:
Thanks this looks much better